frontend/msSmash.pas

```diff
@@ -134,8 +133,8 @@ procedure AddRequiredMasters(var aFile: IwbFile; const el: IwbElement);
           aFile.CleanMasters;
 
         aFile.AddMasters(slMasters);
+        Logger.Write('PATCH', 'MASTERS', 'Added masters: ' + slMasters.CommaText);
```

Comment:
:+1:

This is nice because it'll allow us to keep better track of the masters that are added during the smashing process.  An issue was reported to me the other day where the resulting smashed patch was missing a required master, so hopefully this can help debug that.